### ISSUE TITLE
"Expected indentation of NaN spaces"

### ISSUE THREAD

**Issue by Trott** [eslint/eslint#8552]
**Tell us about your environment**

* **ESLint Version:** v4.0.0-alpha.2
* **Node Version:** 7.10.0
* **npm Version:** 4.2.0

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

No configuration file used. `--no-eslintrc` flag passed on CLI.

```
```

**What did you do? Please include the actual source code causing the issue.**

Command run was: `eslint --no-eslintrc test.js`. Contents of `test.js`:

```js
/* eslint indent: [2, 2, {ArrayExpression: first, CallExpression: {arguments: first}, FunctionDeclaration: {parameters: first}, FunctionExpression: {parameters: first},  MemberExpression: off, ObjectExpression: first, SwitchCase: 1}] */

function rtfEscape(string) {
  return string
    .replace(/[\\{}]/g, function(m) {
      return '\\' + m;
    })
    .replace(/\t/g, function() {
      return '\\tab ';
    })
    // eslint-disable-next-line no-control-regex
    .replace(/[\x00-\x1f\x7f-\xff]/g, function(m) {
      return '\\\'' + toHex(m.charCodeAt(0), 2);
    })
    .replace(/\ufeff/g, '')
    .replace(/[\u0100-\uffff]/g, function(m) {
      return '\\u' + toHex(m.charCodeAt(0), 4) + '?';
    });
}
```

**What did you expect to happen?**

Any indentation expectations would be reported as an integer.

**What actually happened? Please include the actual, raw output from ESLint.**

Indentation expectations reported as `NaN`:

```console
   6:1  error  Expected indentation of NaN spaces but found 6  indent
   7:1  error  Expected indentation of NaN spaces but found 4  indent
   9:1  error  Expected indentation of NaN spaces but found 6  indent
  10:1  error  Expected indentation of NaN spaces but found 4  indent
  13:1  error  Expected indentation of NaN spaces but found 6  indent
  14:1  error  Expected indentation of NaN spaces but found 4  indent
  17:1  error  Expected indentation of NaN spaces but found 6  indent
  18:1  error  Expected indentation of NaN spaces but found 4  indent
```

**Comment by not-an-aardvark** [comment]
~~Personally, I indent all my code with NaN spaces.~~

Oops, this is definitely a bug. Thanks for reporting.
